submissions/kirill8210/popup-task/main.css

```diff
@@ -0,0 +1,265 @@
+html{
+    box-sizing: border-box;
+}
+
+*, *::before, *::after{
+    box-sizing: inherit;
+}
+
+body{
+    font-family: Roboto, sans-serif;
+    line-height: 1.3;
+    font-weight: 400;
+}
+
+img{
+    max-width: 100%;
+    height: auto;
+}
+
+button{
+    cursor: pointer;
+    padding: 0;
+}
+
+a{
+    text-decoration: none;
+    color: inherit;
+}
+
+ul{
+    list-style: none;
+    padding: 0;
+    margin: 0;
+}
+
+h1, h2, h3, p{
+    margin: 0;
+}
+
+.container{
+    min-height: 100vh;
+    max-width: 1200px;
+    margin: 0 auto;
+    padding: 0 30px;
+}
+
+.header{
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+}
+
+.menu{
+    display: flex;
+    align-items: center;
+    padding: 10px 0;
+    position: relative;
+}
+
+.menu_li{
+    display: flex;
+    align-items: center;
+    margin-left: 10px;
+}
+
+.menu_li{
+    border: 1px solid transparent;
+    height: 35px;
+}
+
+.menu_li > a:hover{
+    border: 1px solid #e5e5e5;
+    padding: 5px 6px;
+}
+
+.menu_li a:focus {
+    border: 2px solid #a6c8ff;
+}
+
+.menu_li > a:hover:focus {
+    padding: 5px;
+    border: 2px solid #a6c8ff;
+}
+
+.menu_li > a{
+    display: flex;
+    align-items: center;
+    height: 35px;
+    padding: 5px;
+    border: 2px solid transparent;
+}
+
+.menu_li > .checkbox{
+    width: 35px;
+    height: 35px;
+}
+
+.menu_li > label{
+    display: flex;
+    align-items: center;
+    width: 35px;
+    height: 35px;
+    padding: 5px;
+    border: 1px solid transparent;
+}
+
+.menu_li > .checkbox:hover{
+    border: 1px solid #e5e5e5;
+}
+
+.menu_li > .checkbox:focus {
+    border: 2px solid #a6c8ff;
+}
+
+.menu_li > .checkbox:hover:focus {
+    border: 2px solid #a6c8ff;
+}
+
+.checkbox, .checkbox_more{
+    appearance: none;
+    position: absolute;
+}
+
+.checkbox:checked ~ .popup, .checkbox_more:checked ~ .popup_more{
+    display: flex;
+}
+
+.checkbox:checked ~ label{
+    border: 2px solid #a6c8ff;
+}
+
+.popup_img{
+    width: 25px;
+}
+
+.popup{
+    display: none;
+    position: absolute;
+    top: 50px;
+    right: 0;
+    width: 310px;
+    height: 453px;
+    overflow: auto;
+    flex-flow: wrap;
+    gap: 0 10px;
+    box-shadow: 0 2px 8px rgba(0, 0, 0, .25);
+}
+
+.popup a, .popup_more label{
+    font-size: 14px;
+    line-height: 16px;
+}
+
+.popup img, .popup_more img{
+    max-width: 45px;
+    max-height: 45px;
+    margin: 10px auto 0;
+}
+
+.popup p, .popup_more p{
+    margin: 10px auto;
+}
+
+.popup_li{
+    border: 1px solid transparent;
+    width: 90px;
+    height: 90px;
+    margin-bottom: 10px;
+}
+
+.popup_li > a{
+    border: 2px solid transparent;
+}
+
+.popup_li a:hover{
+    border: 1px solid #e5e5e5;
+}
+
+.popup_li a:hover p{
+    margin-bottom: 11px;
+}
+
+.popup_li a:focus{
```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)

Reply:
ok